gamemode/modules/base/sv_gamemode_functions.lua

```diff
@@ -94,6 +94,18 @@ function GM:canSeeLogMessage(ply, message, colour)
     return true
 end
 
+function GM:canEarnNPCMoney(ply, npc)
+    return GAMEMODE.Config.npckillpay > 0
```

Comment:
Shouldn't this take KillValue into account?

Reply:
I figured that someone wanting all NPCs to be worth $0 except certain ones would be so obscure that anyone that did it could just return true in a hook